src/Microsoft.AspNetCore.Razor.Tools/ServerProtocol/ServerLogger.cs

```diff
@@ -46,7 +46,7 @@ static ServerLogger()
                     // Otherwise, assume that the environment variable specifies the name of the log file.
                     if (Directory.Exists(loggingFileName))
                     {
-                        loggingFileName = Path.Combine(loggingFileName, $"server.{loggingFileName}.{GetCurrentProcessId()}.log");
+                        loggingFileName = Path.Combine(loggingFileName, $"razorserver.{GetCurrentProcessId()}.log");
```

Comment:
This change lets me specify absolute path for directories here.

Reply:
ok